File: exam-slammr-webapp/src/test/java/uk/co/claritysoftware/exam/slammr/webapp/persistence/dynamodb/item/question/QuestionItemTest.java
package uk.co.claritysoftware.exam.slammr.webapp.persistence.dynamodb.item.question;

import org.junit.Test;

import nl.jqno.equalsverifier.EqualsVerifier;

/**
 * Unit test class for {@link QuestionItem}
 */
public class QuestionItemTest {

	@Test
	public void shouldHonourEqualsHashcodeContract() {
		EqualsVerifier.forClass(QuestionItem.class)
				.verify();
	}

}

File: exam-slammr-webapp/src/test/kotlin/uk/co/claritysoftware/exam/slammr/webapp/persistence/dynamodb/item/question/QuestionItemTest.kt
package uk.co.claritysoftware.exam.slammr.webapp.persistence.dynamodb.item.question

import nl.jqno.equalsverifier.EqualsVerifier
import org.junit.Test

/**
 * Unit test class for [QuestionItem]
 */
class QuestionItemTest {

	@Test
	fun shouldHonourEqualsHashcodeContract() {
		EqualsVerifier.forClass(QuestionItem::class.java)
				.verify()
	}

}